gateway/mw_external_oauth.go

```diff
@@ -241,6 +241,11 @@ func (k *ExternalOAuthMiddleware) introspection(accessToken string) (bool, strin
 		}
 	} else {
 		log.WithError(err).Debug("Found OAuth introspection result in the redis cache")
+
+		exp := claims["exp"].(float64)
```

Comment:
Do we need to add ExpiresAtValidationSkew, like in JWT middleware?

Reply:
It wasn’t in the contract and RFC so it is not implemented.